test/messagingsession/DefaultMessagingSession.test.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import { GetMessagingSessionEndpointCommand } from '@aws-sdk/client-chime-sdk-messaging';
```

Comment:
I think we can add unit tests to test:
1. Checking the MessagingSessionConfiguration has a valid endpointUrl if passed as `undefined`. As the startConnecting should set one.
2. In reconnect unit tests, add a unit test to see whether we can mock a chimeClient to send a new endpointUrl and assert that code path works.

Reply:
Will do 2, as stated above we don't want to set value on MessagingSessionConfiguration so won't do 1